torch/_inductor/codegen/aoti_runtime/implementation.cpp

```diff
@@ -36,8 +36,8 @@ void convert_handle_to_arrayref_tensor(
   AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_get_data_ptr(handle, &data_ptr));
   int64_t dim;
   AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_get_dim(handle, &dim));
-  int64_t numel;
-  AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_get_numel(handle, &numel));
+  int64_t storage_numel;
+  AOTI_TORCH_ERROR_CODE_CHECK(aoti_torch_get_storage_numel(handle, &storage_numel));
```

Comment:
Calling `aoti_torch_get_storage_numel` in the generated code will cause a FC breakage in fbcode. Because the service platform update is in a slower pace, your change to `shim_common.cpp` will only be picked up later. In this case we need to split this PR to land in 2 steps.

Try to import this PR to fbcode, if our FC tests don't catch it, we should improve the test coverage, cc @ColinPeppler